symengine/lib/symengine_wrapper.pyx

```diff
@@ -1684,7 +1752,11 @@ class NegativeInfinity(Number):
         return True
 
     def __new__(cls):
-        return -oo
+        cdef Basic r = Number.__new__(Infinity)
+        cdef Number s = Number.__new__(Integer)
+        s.thisptr = <RCP[const symengine.Basic]>symengine.integer(-1)
+        r.thisptr = symengine.make_rcp_Infty(s.thisptr)
+        return r
```

Comment:
We don't do `-oo = c2py(symengine.NegInf)` for negative infinity like we do `oo = c2py(symengine.Inf)` for infinity.